src/openrct2/world/Park.cpp

```diff
@@ -160,6 +162,20 @@ namespace OpenRCT2::Park
         }
 
         suggestedMaxGuests = std::min<uint32_t>(suggestedMaxGuests, 65535);
+
+#ifdef ENABLE_SCRIPTING
+        auto& hookEngine = GetContext()->GetScriptEngine().GetHookEngine();
+        if (hookEngine.HasSubscriptions(HOOK_TYPE::PARK_CALCULATE_GUEST_CAP))
+        {
+            auto ctx = GetContext()->GetScriptEngine().GetContext();
+            auto obj = DukObject(ctx);
+            obj.Set("suggestedGuestMaximum", gameState.SuggestedGuestMaximum);
```

Comment:
```suggestion
            obj.Set("suggestedGuestMaximum", suggestedMaxGuests);
```
To use the actual calculated value, instead of the old on from 13 seconds ago.

Reply:
Sorry I missed that when moving the hook down.